Neos.Neos/Resources/Private/Templates/Module/Management/Workspaces/Show.html

```diff
@@ -180,15 +203,36 @@
+						<i class="fas fa-trash-alt icon-white"></i>
+						{neos:backend.translate(id: 'workspaces.discardAllChanges', source: 'Modules', package: 'Neos.Neos')}
+					</button>
+					<button form="postHelper" formaction="{f:uri.action(action: 'publishWorkspace', arguments: {workspace: selectedWorkspace})}" type="submit" class="review-button-action neos-button neos-button-success">
+						<i class="fas fa-check-double icon-white"></i>
+						{neos:backend.translate(id: 'workspaces.publishAllChangesTo', source: 'Modules', package: 'Neos.Neos', arguments: {0: baseWorkspaceLabel})}
+					</button>
+					<button type="submit" name="moduleArguments[action]" value="discard" class="neos-button neos-button-danger neos-hidden neos-disabled batch-action" disabled="disabled" data-toggle="modal" href="#discardSelected">
+						<i class="fas fa-trash-alt icon-white"></i>
+						{neos:backend.translate(id: 'workspaces.discardSelectedChanges', source: 'Modules', package: 'Neos.Neos')}
+					</button>
+					<button type="submit" name="moduleArguments[action]" value="publish" class="neos-button neos-button-success neos-hidden neos-disabled batch-action" disabled="disabled">
+						<i class="fas fa-check icon-white"></i>
+						{neos:backend.translate(id: 'workspaces.publishSelectedChangesTo', source: 'Modules', package: 'Neos.Neos', arguments: {0: baseWorkspaceLabel})}
+					</button>
 				</f:then>
 				<f:else>
-					<button type="submit" name="moduleArguments[action]" value="publish" class="neos-button neos-hidden neos-disabled batch-action" disabled="disabled">{neos:backend.translate(id: 'workspaces.publishSelectedChanges', source: 'Modules', package: 'Neos.Neos')}</button>
-					<button type="submit" name="moduleArguments[action]" value="discard" class="neos-button neos-hidden neos-disabled batch-action" disabled="disabled">{neos:backend.translate(id: 'workspaces.discardSelectedChanges', source: 'Modules', package: 'Neos.Neos')}</button>
-					<button class="neos-button neos-button-danger" data-toggle="modal" href="#discard">{neos:backend.translate(id: 'workspaces.discardAllChanges', source: 'Modules', package: 'Neos.Neos')}</button>
+					<button type="submit" name="moduleArguments[action]" value="discard" class="neos-button neos-button-danger neos-hidden neos-disabled batch-action" disabled="disabled" data-toggle="modal" href="#discardSelected">
+						<i class="fas fa-trash-alt icon-white"></i>
+						{neos:backend.translate(id: 'workspaces.discardSelectedChanges', source: 'Modules', package: 'Neos.Neos')}
+					</button>
+					<button type="submit" name="moduleArguments[action]" value="publish" class="neos-button neos-button-success neos-hidden neos-disabled batch-action" disabled="disabled">
+						<i class="fas fa-check icon-white"></i>
+						{neos:backend.translate(id: 'workspaces.publishSelectedChangesTo', source: 'Modules', package: 'Neos.Neos', arguments: {0: baseWorkspaceLabel})}
+					</button>
```

Comment:
As this button will be removed with #4021 on 7.3 wouldn't it be easier for an upmerge if we remove the button here as well? This would leave both sides with two buttons inside `f:else`.